// Distributed under GPLv3 only as specified in repository's root LICENSE file

syntax="proto2";

package tag.aas;

message MediaChannelSetupResponse
{
    required uint32 unknown_field_1 = 1;
    required uint32 max_unacked = 2;
    repeated uint32 configs = 3;
}
